ui/components/src/ContentWithLegend/model/content-with-legend-model.test.ts

```diff
@@ -36,118 +37,124 @@ describe('getContentWithLegendLayout', () => {
     });
   });
 
```

Comment:
I put a loop around these tests to check both list and table, which made the diff a little funky, but the tests haven't changed much since the previous PR.